cogeo_mosaic/model.py

```diff
@@ -24,3 +31,169 @@ class MosaicJSON(BaseModel):
     bounds: List[float] = Field([-180, -90, 180, 90])
     center: Optional[Tuple[float, float, int]]
     tiles: Dict[str, List[str]]
+
+    @classmethod
+    def from_dict(cls, mosaic_def: Dict):
+        return MosaicJSON(**dict(mosaic_def))
```

Comment:
if it's already a dict you don't need `dict(mosaic_def)`?

Reply:
I know, but it doesn't hurt to coerce

Reply:
well it just doesn't make sense to me to re-dict a dict ;-)